doc/modules/mod_pubsub.md

```diff
@@ -118,6 +118,45 @@ This node type **requires** `publish-options` with at least `device_id` and `ser
 
 If you'd like to learn more about metrics in MongooseIM, please visit the [MongooseIM metrics](../operation-and-maintenance/Mongoose-metrics.md) page.
 
+#### Overall PubSub action metrics
+
+For every PubSub action, like node creation, subscription, publication there are following kind of metrics:
+
+* count - a spiral metric measuring number of given action invocations
```

Comment:
spiral or circular?
count - a spiral metric showing the number of given action invocations
count - a spiral metric showing the number of given action's invocations
count - a spiral metric showing the number of invocations of the given action